package.json

```diff
@@ -1,840 +1,841 @@
+						"scope": "window"
+					},
+					"maven.pomfile.prefetchEffectivePom": {
+						"type": "boolean",
+						"default": false,
+						"description": "%configuration.maven.pomfile.prefetchEffectivePom%",
+						"scope": "application"
+					},
+					"maven.dependency.enableConflictDiagnostics": {
+						"type": "boolean",
+						"default": "true",
+						"description": "%configuration.maven.dependency.enableConflictDiagnostics%",
+						"scope": "window"
+					},
+					"maven.projectOpenBehavior": {
+						"default": "Interactive",
+						"type": "string",
+						"scope": "window",
+						"description": "%configuration.maven.projectOpenBehavior%",
+						"enum": [
+							"Interactive",
+							"Open",
+							"Add to Workspace",
+							"None"
+						]
+					},
+					"maven.explorer.projectName": {
+						"default": "${project.name}",
+						"type": "string",
+						"scope": "window",
+						"description": "%configuration.maven.explorer.projectName%",
+						"enum": [
+							"${project.name}",
+							"${project.artifactId}",
+							"${project.artifactId}-${project.version}",
+							"${project.groupId}.${project.artifactId}",
+							"${project.groupId}.${project.artifactId}-${project.version}"
+						]
+					}
+				}
+			}
+		]
+	},
+	"scripts": {
+		"vscode:prepublish": "webpack --mode production",
+		"build-plugin": "node scripts/build-jdtls-ext.js",
+		"compile": "tsc -p ./",
+		"tslint": "tslint -t verbose --project tsconfig.json",
+		"watch": "webpack --mode development --watch --progress",
+		"test": "tsc -p ./ && node ./out/test/index.js",
+		"webpack": "webpack --mode development",
+		"update-tpn": "node scripts/update-third-party-notice.js",
+		"update-xsd": "tsc scripts/generate-maven-xsd.ts && node scripts/generate-maven-xsd.js"
+	},
+	"devDependencies": {
+		"@types/fs-extra": "^4.0.9",
+		"@types/glob": "^7.2.0",
+		"@types/lodash": "^4.14.186",
+		"@types/lru-cache": "^5.1.0",
+		"@types/md5": "^2.3.2",
+		"@types/minimatch": "^3.0.4",
+		"@types/mocha": "^9.1.1",
+		"@types/node": "^16.18.3",
+		"@types/semver": "^7.3.12",
+		"@types/vscode": "1.65.0",
+		"@types/which": "^1.3.2",
+		"@types/xml2js": "^0.4.11",
+		"@vscode/test-electron": "^2.1.5",
+		"glob": "^7.2.3",
+		"mocha": "^9.2.2",
+		"ts-loader": "^9.4.1",
+		"tslint": "^6.1.3",
+		"typescript": "^4.8.4",
+		"webpack": "^5.74.0",
+		"webpack-cli": "^4.10.0"
+	},
+	"dependencies": {
+		"domhandler": "^5.0.3",
+		"expand-home-dir": "0.0.3",
+		"fast-glob": "^3.2.12",
+		"fs-extra": "^4.0.3",
+		"get-port": "^4.2.0",
+		"htmlparser2": "^8.0.1",
+		"jszip": "^3.10.1",
+		"lodash": "^4.17.21",
+		"lru-cache": "^6.0.0",
+		"md5": "^2.2.1",
+		"minimatch": "^3.1.2",
+		"semver": "^7.3.8",
+		"vscode-extension-telemetry-wrapper": "^0.13.2",
+		"vscode-languageclient": "^7.0.0",
+		"vscode-languageserver-protocol": "^3.17.2",
+		"vscode-tas-client": "^0.1.63",
+		"which": "^1.3.1",
+		"xml2js": "^0.4.23"
+	},
+	"__metadata": {
```

Comment:
the indentation of `package.json ` has been changed, and extra `__metadata` section is added, perhaps formatted by the test library when you run `npm test`. 

Can you remove this section and change the indentation back?

Reply:
All the other changes LGTM.

Reply:
Oh forget about this, let me do it, I will also add `zh` translation.